baseline/client/flanker/frag/instruction.html

```diff
@@ -0,0 +1,5 @@
+<h1>Instruction:</h1>
+For each trial in this task, 5 arrows will be displayed on the screen. <br>
+Your goal is to press the button that corresponds to the direction of the <i>center arrow</i>. <br>
```

Comment:
Very, very minor nit: These days <em> is generally preferred to <i> for this sort of usage. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em